Rebuild-Black-Business/RBB-Website
# Change logo to SVG for better resolution

Per issue #212, the logo pulled in from Cloudinary looked really
pixelated. I changed it to an SVG, implemented through the `Logo`
component found at `src/components/SVG/Logo.js`.

Fixes #212 

## Pages/Interfaces that will change

Will apply to all pages that contain the `PrimaryNav` component

### Before

![image](https://user-images.githubusercontent.com/45179933/84434680-1a505c00-abf6-11ea-9130-2f9e5829ac9b.png)

![image](https://user-images.githubusercontent.com/45179933/84434653-115f8a80-abf6-11ea-826d-5eca243ff2d1.png)

### After

![image](https://user-images.githubusercontent.com/45179933/84434582-eb39ea80-abf5-11ea-8c9f-67ea95bd79a3.png)

![image](https://user-images.githubusercontent.com/45179933/84434559-df4e2880-abf5-11ea-9012-2d2d14a999ab.png)

## Steps to test

1. Ran `npm run test` and all tests passed
2. Did a check in the inspector on different sizes

### Additional notes
This is my first PR, so feel free to discard if not useful/helpful.
